src/flavors/transportation-demo/config.json

```diff
@@ -4,7 +4,7 @@
     "name": "Regional Transportation Plan",
```

Comment:
Note that all changes to flavor config files in this PR are just to remove the gettext markup.